test/test_jing.rb

```diff
@@ -113,6 +113,14 @@ def test_successful_exit_status_with_unparsable_output_raises_an_exception
     end
   end
 
+  def test_java_options_env
+    java_opts = ENV["_JAVA_OPTIONS"]
+    ENV["_JAVA_OPTIONS"] = "-Djava.io.tmpdir=#{Dir.tmpdir}"
+    errors = Jing.new(RNG_SCHEMA).validate(VALID_XML)
```

Comment:
Maybe I'm missing something bug shouldn't this be ran under `fakeshell`, setting `:output` to something that contains the warning message?

Reply:
Totaly make sense, done